src/classes/UTIL_Namespace.cls

```diff
@@ -45,6 +45,7 @@ public class UTIL_Namespace {
     * @description Finds the namespace for the current context.
     * @return string The current namespace as a string, or a blank string if we're not in a namespaced context.
     ********************************************************************************************************/
+    @AuraEnabled
```

Comment:
@force2b - Do we have a policy of avoiding AuraEnabled in our util classes? Should we support this with a pass-through method in the LWC component controller?

Reply:
I'm not sure there's an official policy, but I wouldn't recommend it. Your suggestion of a pass through would be better.